docs/concepts/validation_decorator.md

```diff
@@ -392,31 +320,16 @@ print(foo())
 
 ### Validation exception
 
-Currently upon validation failure, a standard Pydantic [`ValidationError`][pydantic_core.ValidationError] is raised.
-See [model error handling](models.md#error-handling) for details.
-
-This is helpful since its `str()` method provides useful details of the error which occurred and methods like
-`.errors()` and `.json()` can be useful when exposing the errors to end users. However, `ValidationError` inherits
-from `ValueError` **not** `TypeError`, which may be unexpected since Python would raise a `TypeError` upon invalid
-or missing arguments. This may be addressed in future by either allowing a custom error or raising a different
-exception by default, or both.
-
-### Coercion and strictness
-
-Pydantic currently leans on the side of trying to coerce types rather than raise an error if a type is wrong,
-see [model data conversion](models.md#data-conversion) and `@validate_call` is no different.
+Currently upon validation failure, a standard Pydantic [`ValidationError`][pydantic_core.ValidationError] is raised
+(see [model error handling](models.md#error-handling) for details). This is also true for missing required arguments,
+where Python normally raises a [`TypeError`][].
 
 ### Performance
 
-We've made a big effort to make Pydantic as performant as possible
-and argument inspect and model creation is only performed once when the function is defined, however
-there will still be a performance impact to using the `@validate_call` decorator compared to
-calling the raw function.
-
-In many situations this will have little or no noticeable effect, however be aware that
-`@validate_call` is not an equivalent or alternative to function definitions in strongly typed languages;
-it never will be.
-
-### Return value
+We've made a big effort to make Pydantic as performant as possible. While the inspection of the decorated
+function is only performed once, there will still be a performance impact when making calls to the function
+compared to using the original function.
 
-The return value of the function may optionally be validated against its return type annotation.
+In many situations, this will have little or no noticeable effect, however be aware that
+[`validate_call()`][pydantic.validate_call] is not an equivalent or alternative to function
+definitions in strongly typed languages; it never will be.
```

Comment:
```suggestion
In many situations, this will have little or no noticeable effect. However, be aware that
[`validate_call()`][pydantic.validate_call] is not an equivalent or alternative to function
definitions in strongly typed languages, and it never will be.
```